tka/state.go

```diff
@@ -205,6 +245,7 @@ func (s State) applyVerifiedAUM(update AUM) (State, error) {
 const (
 	maxDisablementSecrets = 32
 	maxKeys               = 512
+	maxDenylistEntries    = 64
```

Comment:
I feel like this makes the node deletion flow kind of awkward, in that sometimes you'll hit the deny list limit and have to rotate your signing key, and in the event that there's more than one signing key, that might mean being forced to revoke one of the other signing keys. I'm not sure how to do that without making the node creation flow more burdensome.

Reply:
Yeah, I haven't really thought through how this works from a UX perspective. Im thinking that even if we do decide to merge this, we should probably keep this feature fairly hidden and only mention it to users who are stuck in a bind & use this primitive.

I pulled the `64` entries limit out of thin air: It corresponds to about 2.5kb of data. Perhaps we can bump it up much higher?

Reply:
I think any fixed limit such that the number of free entries is less than the number of devices in the tailnet is going to have limited utility. I guess 64 is big enough to make it useful for an emergency push for a handful of devices being compromised; anything bigger than that and you really need to rotate the signing keys + re-sign everything.